Update kmm-plugin-releases.md (#2788)
JetBrains/kotlin-web-site: docs/topics/kmm/kmm-plugin-releases.md

<tr>
<td>

**0.3.1**

Released: 14 February, 2022

</td>
<td>

> * [Enable M1 iOS simulator in KMM wizards](https://youtrack.jetbrains.com/issue/KT-51105).

Reviewer: ```suggestion
* [Enabled M1 iOS simulator in Kotlin Multiplatform Mobile wizards](https://youtrack.jetbrains.com/issue/KT-51105).
```

* Improve perfomance for indexing XcProjects [KT-49777](https://youtrack.jetbrains.com/issue/KT-49777), [KT-50779](https://youtrack.jetbrains.com/issue/KT-50779).

> * Clean up build scripts: use `kotlin("test")`.

Reviewer: ```suggestion
* Build scripts clean up: use `kotlin("test")` instead of kotlin("test-common") and kotlin("test-annotations-common").
```

* Increase compatibility range with [Kotlin plugin version](https://youtrack.jetbrains.com/issue/KTIJ-20167).

> * [Fix problem with JVM debug on Windows host](https://youtrack.jetbrains.com/issue/KT-50699).

Reviewer: ```suggestion
* [Fixed the problem with JVM debug on Windows host](https://youtrack.jetbrains.com/issue/KT-50699).
```

> * [Fix problem with invalid version after disabling the plugin](https://youtrack.jetbrains.com/issue/KT-50966).

Reviewer: ```suggestion
* [Fixed the problem with the invalid version after disabling the plugin](https://youtrack.jetbrains.com/issue/KT-50966).
```

</td>
<td>

> * [Kotlin 1.5.0](releases.md#release-details)

Reviewer 2: I think that it will be better to mention both compatible versions of the Kotlin plugin so everyone clearly understands that they can use the KMM plugin with them, e.g. 1.5.0 and 1.6.0.

> * [Kotlin 1.5.0](releases.md#release-details)

Reviewer 3: I think that it will be better to mention both compatible versions of the Kotlin plugin so everyone clearly understands that they can use the KMM plugin with them, e.g. 1.5.0 and 1.6.0.

> * [Kotlin 1.5.0](releases.md#release-details)

Reviewer: ```suggestion
* [Kotlin 1.5.0—1.6.*](releases.md#release-details)
```
